release-team/role-handbooks/patch-release-manager/README.md

```diff
@@ -1,41 +1,97 @@
+of support in terms of patches for bugfixes and ongoing CI insuring
+the branch's health and the ability to update from 1.X.Y to 1.(X+1).Y,
+for the newest Y on each of those two branches.
+
+Specific duties of the Patch Release Manager include:
+
+- Ensuring the release branch (e.g. `release-1.13`) remains in a
+  healthy state as measured by the branch version specific release-blocking
+  CI (e.g. for version 1.13 it is [testgrid sig-release-1.13-blocking
+  board](https://testgrid.k8s.io/sig-release-1.13-blocking). If the
+  build breaks or any CI for the release branch becomes unhealthy due
+  to a bad merge or infrastructure issue, ensure that actions are
+  taken ASAP to bring it back to a healthy state.
+- Reviewing and approving [cherry
+  picks](https://git.k8s.io/community/contributors/devel/cherry-picks.md) to
+  the release branch.
+  - Patch releases should not contain new features, so ensure that
+    cherry-picks are for bug/security fixes only.
+  - Cherry picks should not destabilize the branch, so ensure that
+    either the PR has had time to stabilize in master or will have
+    time to stabilize in the release branch before the next patch
+    release is cut.
+- Setting the [schedule and cadence for patch
+  releases](/releases/patch-releases.md) and cutting the
+  [releases](https://github.com/kubernetes/kubernetes/releases).
+
+While this playbook is intended to guide Patch Release Managers,
+it largely consists of opinions and recommendations from former
+patch release managers.  Each Patch Release Managers is ultimately
+responsible for carrying out their duties in the manner they deem
+best for the project.  The playbook then is more what you might
+call "guidelines" than actual hard rules.  Still each Patch Release
+Manager should endeavor to keep this document up to date, improve
+its content, and improve the overall process of patch management
+for the project.
 
 ## Getting started
 
-* Add yourself to the [Release Manager table](/release-managers.md)
-  so the community knows you're the point of contact.
-* Ask a maintainer to add you to the [kubernetes-release-managers](https://github.com/orgs/kubernetes/teams/kubernetes-release-managers/members)
+You're here because you've volunteered and have been selected to
+contribute to patch release management for one or more branches.
+You will need to make yourself known to the community and gain
+access to multiple build and release tools:
+
+* Add your name and contact info, and if applicable a new section
+  for the current release, to the [Patch Releases landing
+  page](/releases/patch-release.md) so the community knows you're the
+  point of contact.  If your work is specific to one particular
+  release, add your name to that release's release team in the patch
+  release manager row, for example [here for
+  1.13](https://git.k8s.io/sig-release/releases/release-1.13/release_team.md).
+* Ask a GitHub team maintainer to add you to the
```

Comment:
Can we instead mention peribolos here or to just [send a PR against k/org](https://github.com/kubernetes/org/blob/30348924b23169ae8d73f50d3b71ef7823c6128a/config/kubernetes/sig-release/teams.yaml#L87) to add themselves to the team?

Reply:
related: https://github.com/kubernetes/community/issues/3102

Reply:
Updated